Source/Extensions/Blazorise.Components/Autocomplete.razor.cs

```diff
@@ -1091,6 +1091,9 @@ protected string CssStyle
             if ( Style != null )
                 sb.Append( Style );
 
+            if ( Disabled != false )
+                sb.Append( "background-color: #e9ecef;" );
```

Comment:
Why exactly is this color hardcoded for the disabled state? What will happen when the user has dark-mode, for example in Tailwind?